test/e2e/operator_groups_e2e_test.go

```diff
@@ -2043,6 +2045,169 @@ var _ = Describe("Operator Group", func() {
 		})
 		require.NoError(GinkgoT(), err)
 	})
+	It("OperatorGroupLabels", func() {
+		c := newKubeClient(GinkgoT())
+		crc := newCRClient(GinkgoT())
+
+		// Create the namespaces that will have an OperatorGroup Label applied.
+		testNamespaceA := genName("namespace-a-")
+		testNamespaceB := genName("namespace-b-")
+		testNamespaceC := genName("namespace-c-")
+		testNamespaces := []string{
+			testNamespaceA, testNamespaceB, testNamespaceC,
+		}
+
+		// Create the namespaces
+		for _, namespace := range testNamespaces {
+			_, err := c.KubernetesInterface().CoreV1().Namespaces().Create(&corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: namespace,
+				},
+			})
+			require.NoError(GinkgoT(), err)
```

Comment:
FYI: The basic ginkgo conversion just added a wrapper of `Describe/ It` container around existing tests. The plan is to have a full-fledge BDD ginkgo tests which includes - writing assertions using [gomega](https://onsi.github.io/gomega/) assertions. I'll probably have a document on how to do this conversion so that it is incorporated by the team as they write more e2e tests